tests/ast-alignment/utils.ts

```diff
@@ -194,6 +195,29 @@ export function preprocessBabylonAST(ast: any): any {
           node.params = node.parameters;
           delete node.parameters;
         }
+      },
+      /**
+       * We want this node to be different
+       * @see https://github.com/JamesHenry/typescript-estree/issues/109
+       */
+      TSTypeParameter(node: any) {
```

Comment:
i was able to transform babel node to `Identifier`, and we still can do alignment tests for other parts of code